src/crud_internal.c

```diff
@@ -119,188 +124,174 @@ int createObject( wrp_msg_t *reqMsg , wrp_msg_t **response)
 		}
 	}
 
-	if(reqMsg->u.crud.dest !=NULL)
+	if(jsonData !=NULL)
 	{
-		destVal = strdup(reqMsg->u.crud.dest);
-		ParodusInfo("destVal is %s\n", destVal);
-
-		if( (destVal != NULL))
-		{
-			child_ptr = strtok(destVal , "/");
+		free( jsonData );
+		jsonData = NULL;
+	}
 
-			/* Get the 1st object */
-			obj[0] = strdup( child_ptr );
-			ParodusPrint( "parent is %s\n", obj[0] );
+	if(reqMsg->u.crud.dest !=NULL)
+	{
+		ParodusInfo("reqMsg->u.crud.dest is %s\n", reqMsg->u.crud.dest);
+		child_ptr = strtok(reqMsg->u.crud.dest , "/");
 
-			free(destVal);
+		/* Get the 1st object */
+		obj[0] = strdup( child_ptr );
+		ParodusPrint( "parent is %s\n", obj[0] );
 
-			while( child_ptr != NULL)
+		while( child_ptr != NULL)
```

Comment:
This can overflow the hard coded limit of 5 above.  Needs to be limited to same max.

Reply:
dest parsing logic is modified to use wrp_get_msg_dest_element() and hence this overflow problem is solved.